librad/src/net/protocol/membership/periodic.rs

```diff
@@ -56,7 +57,17 @@ where
         }
     });
 
-    if let Err(e) = stream::select(shuffle, promote).map(Ok).forward(tx).await {
+    let tickle = Interval::new(Duration::from_secs(15), Duration::from_secs(5))
```

Comment:
I think we will need to ensure that this value is always smaller than `net::quic::MAX_IDLE_TIMEOUT`. I know this kind of crosses domains, but perhaps we could just say `MAX_IDLE_TIMEOUT / 2`?

Reply:
Good catch, was inclined to expose another param similar to promotion and shuffle. Given that this is easy to misconfigure and has a strong relation to other consts in the system For that I'd not expose and turn to the scientif method of half of the max as well.